system_config:
  ros__parameters:
    simulation_mode: false,
    system_frequency: 60  # Speed of hot loop (Hertz)
    parking_lot: false  # Parking lot test. Drives straight slowly for 5 seconds then stops.
    speed: 5  # %
    steering: 0  # degrees